docs/common_links.rst

```diff
@@ -7,6 +7,7 @@
 
 .. |ClassicalTransport| replace:: :class:`~plasmapy.formulary.braginskii.ClassicalTransport`
 .. |RelativisticBody| replace:: :class:`~plasmapy.formulary.relativity.RelativisticBody`
+.. |SingleParticleCollisionFrequencies| replace:: :class:`~plasmapy.formulary.collisions.frequencies.SingleParticleCollisionFrequencies`
```

Comment:
This might be our longest namespace yet!

Reply:
Have we thought about moving the stuff from `collisions` to `formulary`, btw?

Reply:
I can't remember if we've talked about that.  I've also wondered if it would be appropriate to have `plasmapy.collisions` be its own subpackage, since it is a really big topic and there's a lot more that could go in it beyond what would be appropriate for the formulary.